tests/Reading/PropertyReadMethodsTest.php

```diff
@@ -292,34 +316,39 @@ public function testGetDoubleAndDecimalSame()
     public function testGetDate()
     {
         $date = $this->dateProperty->getDate();
-        $this->assertInstanceOf('DateTime', $date);
+        $this->assertInstanceOf(DateTime::class, $date);
         $this->assertEquals('1303392860', $date->format('U'));
     }
+
     public function testGetDateMulti()
     {
         $multidate = $this->node->getProperty('index.txt/jcr:content/multidate');
-        $this->assertEquals(\PHPCR\PropertyType::DATE, $multidate->getType());
+        $this->assertEquals(PropertyType::DATE, $multidate->getType());
         $arr = $multidate->getValue();
         $this->assertInternalType('array', $arr);
+
         foreach ($arr as $v) {
-            $this->assertInstanceOf('DateTime', $v);
+            $this->assertInstanceOf(DateTime::class, $v);
         }
         //check correct values and sort order
-        $expectedArray = array(
-                new \DateTime('2011-04-22T14:34:20+01:00'),
-                new \DateTime('2011-10-23T14:34:20+01:00'),
-                new \DateTime('2010-10-23T14:34:20+01:00'), );
+        $expectedArray = [
+                new DateTime('2011-04-22T14:34:20+01:00'),
+                new DateTime('2011-10-23T14:34:20+01:00'),
+                new DateTime('2010-10-23T14:34:20+01:00')
```

Comment:
My fault will fix it. :)

Cool! At this time I don't know nothing about open source, unit tests and OOP... In March 2010 I created my very first project - browser game written in procedural PHP with MySQL it was spaghetti code but it was very fast because I spent long time with SQL optimization and PHP micro optimization it was funny :D :D